### DISCUSSION TITLE
Unknown device!

### DISCUSSION THREAD

**Discussion by Jaron-Wilson** [d4rken-org/capod#266] [Q&A]
60 Unknown device 1 second ago This is an unknown device, but it is using similar message format. Let's add support for it, contact me :) Raw data 76: 07 19 01 25 20 00 05 80 06 00 05 67 CA 2F 81 78 02 9D 2A 17 18 F8 3C 80 65 6D 9F

**Comment by d4rken** [comment]
Well, what device could it be? :grin:
